CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Move lz4 Compressor into gocql package (CASSGO-32)
```

Comment:
I think it should be something like `Move lz4 compressor to lz4 package within the gocql module`